FinSearchUnified/Tests/Bundle/StoreFrontBundle/Gateway/Findologic/Hydrator/CustomListingHydratorTest.php

```diff
@@ -145,6 +145,20 @@ public function facetFilterProvider()
                 'special_characters',
                 'Sonderzeichen',
                 ProductAttributeFacet::MODE_RADIO_LIST_RESULT
+            ],
+            'Filter with multi-byte characters' => [
+                'filterArray' => [
+                    'type' => 'select',
+                    'name' => 'TISCHWÄSCHE - Ausführung',
+                    'display' => 'Multibyte - TISCHWÄSCHE - Ausführung',
+                    'select' => 'single'
+                ],
+                'expectedName' => 'TISCHWÄSCHE - Ausführung',
+                'expectedUniqueKey' => 'TISCHWÄSCHE - Ausführung',
+                'expectedAttributeName' => 'product_attribute_TISCHWÄSCHE - Ausführung',
+                'expectedAttributeFormFieldName' => 'TISCHW_SCHE_-_Ausführung',
+                'expectedAttributeLabel' => 'Multibyte - TISCHWÄSCHE - Ausführung',
+                'expectedAttributeMode' => ProductAttributeFacet::MODE_RADIO_LIST_RESULT
```

Comment:
The structure is different than with the other data provider cases, is that correct?

Reply:
No I just like to have the names of the parameters for the data provider directly inside of the data provider. This way I can read a data provider without the need of the test method and I can add new cases quicker.

Reply:
Can you update the other cases than to have the same structure?